<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitControl</class>
 <widget class="QWidget" name="FitControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>881</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="actionsLayout">
       <item>
        <widget class="QPushButton" name="pushButtonFit">
         <property name="toolTip">
          <string>Start multi-dataset fitting</string>
         </property>
         <property name="text">
          <string>&amp;Fit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClearModel">
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlModel">
       <item>
        <widget class="QPushButton" name="pbBuiltIn">
         <property name="text">
          <string>Built-in</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCustom">
         <property name="text">
          <string>Custom</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MantidQt::MantidWidgets::FunctionBrowser" name="functionBrowser" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="MantidQt::CustomInterfaces::DynamicPDF::DPDFFitOptionsBrowser" name="fitOptionsBrowser" native="true">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::FunctionBrowser</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/FunctionBrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::DynamicPDF::DPDFFitOptionsBrowser</class>
   <extends>QWidget</extends>
   <header>DPDFFitOptionsBrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
